doc/reference/peripherals/ps2.rst

```diff
@@ -0,0 +1,21 @@
+.. _ps2:
+
+
+PS2
+####
+
+Overview
+********
+
```

Comment:
How about something like this for the Overview:
```
The PS/2 connector first hit the market in 1987 on
IBM's desktop PC line of the same name before
becoming an industry-wide standard for mouse and
keyboard connections. Starting around 2007, USB
superseded PS/2 and is the modern peripheral device
connection standard. For legacy support on boards
with a PS/2 connector, Zephyr provides these PS/2 driver APIs.
```

Reply:
NP, I even updated every occurrence of PS2 in the commit messages.